sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NormalizeFloatingPointNumbersSuite.scala

```diff
@@ -91,5 +91,38 @@ class NormalizeFloatingPointNumbersSuite extends PlanTest {
 
     comparePlans(doubleOptimized, correctAnswer)
   }
+
+  test("directly normalize on the children of If") {
```

Comment:
I guess we can remove `directly` here and line 110.

Reply:
And, maybe, `normalize on the children` -> `normalize the children`?

Reply:
Okay.